config.txt

```diff
@@ -0,0 +1,946 @@
+# Include directories ---------------------------------------------------------
+
+# Clang compiler arguments ----------------------------------------------------
++arg any: -x
++arg any: c++
++arg any: -std=c++17
++arg any: -D__CODE_GENERATOR__
++arg any: -Wno-deprecated-declarations
++arg any: -ferror-limit=0
++arg win32: -fms-compatibility
++arg win32: -fms-extensions
++arg win32: -fms-compatibility-version=19
++arg win32: -fdelayed-template-parsing
++arg linux: -D_INC_WINDOWS
++arg linux: -D__WINDOWS_H_INCLUDED
+
+# Immutable types -------------------------------------------------------------
++immutable Standard_Boolean
++immutable Standard_CString
++immutable Standard_Integer
++immutable Standard_Real
+
+# Excluded classes ------------------------------------------------------------
+-class AIS_LocalStatus
+
+-class BRepMesh_ConeRangeSplitter
+-class BRepMesh_CylinderRangeSplitter
+-class BRepMesh_DefaultRangeSplitter
+-class BRepMesh_SphereRangeSplitter
+-class BRepMesh_TorusRangeSplitter
+-class BRepMesh_UVParamRangeSplitter
+-class BRepMesh_NURBSRangeSplitter
+-class BRepMesh_BoundaryParamsRangeSplitter
+
+-class Cocoa_LocalPool
+
+#-class Font_BRepFont
```

Comment:
after taking out the moving binder part of this PR, might want to rebase on that open PR i have and adjust the config file as needed